src/legacy/components/analytics/hooks/useAnalyticsReporter.ts

```diff
@@ -53,6 +56,12 @@ let initiatedPixel = false
 // tracks web vitals and pageviews
 export function useAnalyticsReporter() {
   const { pathname, search } = useLocation()
+  const tradeTypeInfo = useTradeTypeInfo()
+  const derivedTradeState = useDerivedTradeState()
+
+  const sellSymbol = derivedTradeState?.state?.inputCurrency?.symbol
+  const buySymbol = derivedTradeState?.state?.outputCurrency?.symbol
+  const tradePair = sellSymbol && buySymbol ? `${sellSymbol},${buySymbol}` : null
```

Comment:
Could you refactor this into a hook just to clean up a bit?
For example:

```ts
const tradePair = useTradePair()
```

Reply:
Actually, you could include in the hook the `tradeTypeInfo` and return already the `market` string from this line and make it simpler https://github.com/cowprotocol/cowswap/pull/2930/files#diff-5928508e7db7ab134e4f07b25931f0a7c66990132eed75af23532eb4e90d7a61R105

Reply:
Good suggestion, updated!